.github/workflows/duplicate_sample.yaml

```diff
@@ -0,0 +1,27 @@
+name: Duplicate-Package-Sample
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  duplicate:
+    name: duplicate package sample
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Run duplicate script file
+        run: bash ${GITHUB_WORKSPACE}/duplicate_package_sample.sh
+
+      - name: Commit and Push
+        run: |
+          git config --global user.email "robel@chainsafe.io"
+          git config --global user.name "Robel Getnet"
+          git config --global user.username "robGG1997"
```

Comment:
```suggestion
          git config --global user.email "${{ github.actor }}@users.noreply.github.com"
          git config --global user.name "${{ github.actor }}"
```

Reply:
I think there is no better solution right now. 
However let's personalize commit to the person who initiated workflow to run.

Reply:
done, now the commit author will be whoever initiated the workflow, I've also tested this